packages/odyssey/src/scss/vendors-ext/_choices-ext.scss

```diff
@@ -0,0 +1,453 @@
+
+.ods-select {
+  position: relative;
+  max-width: $max-line-length;
+  overflow: hidden;
+  font-size: ms(0);
+
+  &:focus,
+  &.is-ods-select-focused {
+    border-radius: $base-border-radius;
+    outline: none;
+    box-shadow: 0 0 0 4px fauxpacity(cv('action', 'base'));
+
+    &.is-ods-select-open {
+      box-shadow: 0 0 0 4px fauxpacity(cv('action', 'base')), 0 2px 0 4px fauxpacity(cv('action', 'base'));
+
+      &.is-ods-select-flipped {
+        box-shadow: 0 0 0 4px fauxpacity(cv('action', 'base')), 0 -2px 0 4px fauxpacity(cv('action', 'base'));
+      }
+    }
+  }
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  &.is-ods-select-open {
+    overflow: initial;
+  }
+
+  &.is-ods-select-disabled {
+    .ods-select--inner,
+    .ods-select--input {
+      border-color: cv('gray', '200');
+      background: cv('gray', '000');
+      color: $text-sub;
+      cursor: not-allowed;
+      user-select: none;
+
+      &:hover {
+        border-color: cv('gray', '200');
+      }
+    }
+
+    &:focus,
+    &.is-ods-select-focused {
+      box-shadow: none;
+    }
+
+    .ods-select--item {
+      cursor: not-allowed;
+    }
+  }
+
+  [data-invalid] & {
+    &:focus,
+    &.is-ods-select-focused {
+      border-radius: $base-border-radius;
+      outline: none;
+      box-shadow: 0 0 0 4px fauxpacity(cv('danger', 'base'));
+
+      &.is-ods-select-open {
+        box-shadow: 0 0 0 4px fauxpacity(cv('danger', 'base')), 0 2px 0 4px fauxpacity(cv('danger', 'base'));
+
+        &.is-ods-select-flipped {
+          box-shadow: 0 0 0 4px fauxpacity(cv('danger', 'base')), 0 -2px 0 4px fauxpacity(cv('danger', 'base'));
+        }
+      }
+    }
+
+    .ods-select--inner,
+    .ods-select--input {
+      border-color: cv('danger');
+    }
+  }
+
+  [hidden] {
+    display: none !important;
+  }
+}
+
+.ods-select[data-type*='select-one'] {
+  .ods-select--inner {
+    padding-bottom: $tiny-spacing;
+  }
+
+  .ods-select--input {
+    display: block;
+    width: 100%;
+    margin: 0;
+    padding: $small-spacing;
+    border-bottom: 1px solid $base-border-color;
+    background-color: $white;
+  }
+
+  .ods-select--button {
+    position: absolute;
+    top: 50%;
+    right: 0;
+    width: 20px;
+    height: 20px;
+    margin-top: -$small-spacing;
+    margin-right: 25px;
+    padding: 0;
+    border-radius: 10em;
+    opacity: 0.25;
+    background-image: $choices-icon-cross-inverse;
+    background-size: 8px;
+
+    &:hover,
+    &:focus {
+      opacity: 1;
+    }
+
+    &:focus {
+      box-shadow: 0 0 0 2px cv('action', 'base');
+    }
+  }
+
+  .ods-select--item[data-value=''] .ods-select--button {
+    display: none;
+  }
+
+  &::after {
+    content: '';
+    position: absolute;
+    top: 50%;
+    right: $em-small-spacing;
+    width: ms(-1);
```

Comment:
Would this affect if the parent font size is different? Just like the one we saw in link?

Reply:
We don't offer any variant sizing for inputs yet, so this is intentional to match what's available.